tiledb/sm/storage_manager/storage_manager.h

```diff
@@ -1124,13 +1133,14 @@ class StorageManager {
    * Gets the sorted URIs in ascending first timestamp order,
```

Comment:
document the existence of the `timestamp_start` somewhere in this block comment. Mention that the bounds are inclusive.